src/BuiltInTools/dotnet-watch/Internal/ConsoleReporter.cs

```diff
@@ -12,6 +15,17 @@ namespace Microsoft.Extensions.Tools.Internal
     /// </summary>
     public class ConsoleReporter : IReporter
     {
+        private static readonly List<(string prefix, string emoji)> PrefixEmojiAssociations = new()
+        {
+            ("Hot reload", "🔥"),
+            ("HotReload", "🔥"),
+            ("Started", "🚀"),
+            ("Restart", "🔄"),
+            ("Shutdown", "🛑"),
+            ("Building...", "🔧"),
+            ("Error", "❌"),
```

Comment:
Main emoji matching. Additional ones we may want to consider:

- ✏️ for file(s) changed
- ❌ for `Unable to apply hot reload because of a rude edit`. Note this would conflict with `Error`
- ⚒️ for build instead of 🔧
- ➕ / 🆕 for when a new file is added